src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -100,7 +101,15 @@ public async Task RunAsync()
             StepHost.OutputDataReceived += OnDataReceived;
             StepHost.ErrorDataReceived += OnDataReceived;
 
-            string file = Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Externals), "node", "bin", $"node{IOUtil.ExeExtension}");
+            bool useNode10 = ExecutionContext.Variables.GetBoolean("AGENT_USE_NODE10_FOR_NODE_TASKS") ??
```

Comment:
let's make it shorter. `AGENT_USE_NODE10` :D

Reply:
Sure thing!